static/js/components/PersonalTab.js

```diff
@@ -69,6 +78,9 @@ class PersonalTab extends React.Component {
         placeholder="Preferred language"
       /><br />
 
+      <Button raised onClick={this.saveAndContinue.bind(this)}>
+        Save and continue
+      </Button>
```

Comment:
yeah, I think if the editing changes are now in `edit`, which is only created when we start editing, we could gray out the button until the `edit` object has been dispatched to the store - this is a minor UX thing really, that the `save` button visually indicates it won't do anything until you start editing.